### PR TITLE
Refs #21312 - add webpack url with hostname csp

### PR DESCRIPTION
**Pull request by sharvit**
It fixes issue came up with #5184 by adding a second webpack url with hostname to the csp.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -393,6 +397,12 @@ def webpack_server
     @dev_server ||= "#{request.protocol}#{request.host}:#{port}"
   end
 
+  def webpack_server_hostname
+    port = Rails.configuration.webpack.dev_server.port
+    hostname = `hostname`.squish
```
Comment (ohadlevy) [inline app/controllers/application_controller.rb]: can we assume this only happens in development mode?

Comment (sharvit) [reply app/controllers/application_controller.rb]: Yes, it is only happening in development mode.

**Review by amirfefer — CHANGES_REQUESTED** [review]
doesn't work for me, I'm getting this (with `foreman-start-dev`)
```
bootstrap 59b77653f6824f937a87:766 Refused to load the script 'http://0.0.0.0:3808/webpack/react-
intl/locale/en.js' because it violates the following Content Security Policy directive: "script-src 'unsafe-
eval' 'unsafe-inline' 'self' localhost:3808".
```

`foreman start` works as expected.

**Comment by sharvit** [comment]
Closing in favor of #6368

**Pull request closed without merging**